packages/@ember/-internals/metal/tests/namespace_search_test.js

```diff
@@ -1,16 +1,16 @@
-import Mixin from '@ember/object/mixin';
 import { moduleFor, AbstractTestCase } from 'internal-test-helpers';
 
 moduleFor(
   'NamespaceSearch',
   class extends AbstractTestCase {
-    ['@test classToString: null as this inside class must not throw error'](assert) {
-      let mixin = Mixin.create();
-      assert.equal(
-        mixin.toString(),
-        '(unknown mixin)',
-        'this = null should be handled on Mixin.toString() call'
-      );
-    }
+    // TODO: Do we want to test for this case?
```

Comment:
If we want to keep supporting Namespace, we should update this test to not use Mixins.